### PR TITLE
Add properties to answer

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
This is part of a wider piece of work to introduce answer properties to help the quality of the respondent's input.  

Prior to this work, the concept of answer properties was limited to capturing whether or not an answer is 'mandatory'.  This concept has been refactored to support multiple properties and this PR specifically introduces 'decimal' properties.

[Trello Ticket](https://trello.com/c/f978JRq6/402-properties-decimals-1) 

This PR works in conjunction with the following PR's:

API - https://github.com/ONSdigital/eq-author-api/pull/88 
Web App - https://github.com/ONSdigital/eq-author/pull/376 

### How to review 
- Run Tests
- Manually create new answers and ensure UI appears as per attached screenshot.

### CODE REVIEW

**Comment by adammahmood** [comment]
[Properties - decimals (1)](https://trello.com/c/f978JRq6/402-properties-decimals-1)

File: index.js [hunk 1]
```diff
@@ -366,7 +367,7 @@ input UpdateAnswerInput {
   secondaryLabel: String 
   qCode: String
   type: AnswerType
-  mandatory: Boolean
+  properties: JSON
```
Comment (samiwel) [inline index.js]: Will this update all of the answer properties whenever any single property is updated?

Did you consider adding a new mutation to allow properties to be updated independently i.e. 

```
updateProperty(answerId, key, value)
```
and if so what was the reason behind taking this approach?

Comment (adammahmood) [reply index.js]: It updates all properties at once replacing existing JSON with new values. 

Decided not to separate to its own mutation to reduce the code/extra complexity required in the API to handle merging values and to follow existing pattern of mutations. Although, I see the value of smaller updates.  Do you recommend refactoring to handle independent property updates?

File: index.js [hunk 2]
```diff
@@ -60,8 +62,8 @@ interface Answer {
   qCode: String
   label: String
   type: AnswerType!
-  mandatory: Boolean
```
Comment (WickyNilliams) [inline index.js]: Can you keep all `mandatory` fields in place for backwards compatibility (whilst the schema is merged but API/front-end/publisher are not). You can mark as deprecated with the `@deprecated` directive:

```gql
mandatory: Boolean @deprecated(reason: "Use `properties` instead.")
```

File: index.js (hunk 1 above)
Comment (samiwel) [reply index.js]: I don't think it's necessary to refactor right now as there's only a handful of properties. It's something we can change later if we find there is any degradation to performance.

File: index.js (hunk 2 above)
Comment (adammahmood) [reply index.js]: Good idea. Just updated PR.

**Pull request merged**